platform/logging/__tests__/LogLevel.test.ts

```diff
@@ -0,0 +1,117 @@
+import { LogLevel } from '../LogLevel';
```

Comment:
Tests may seem to verbose and redundant, but I thought maybe that's ok for such a core thing :) Let me know if you'd like to simplify them.